content/en/docs/reference/commands/install-cni/index.html

```diff
@@ -702,6 +702,12 @@ <h2 id="envvars">Environment variables</h2>
 <td>If this is set to false, the debug interface will not be enabled, recommended for production</td>
 </tr>
 <tr>
+<td><code>ENABLE_ENHANCED_RESOURCE_SCOPING</code></td>
+<td>Boolean</td>
+<td><code>false</code></td>
+<td>If enabled, meshConfig.discoverySelectors will also limit the configurations(like Gateway,VirtualService,DestinationRule,Ingress, etc)that can be processed by pilot.</td>
```

Comment:
```suggestion
<td>If enabled, meshConfig.discoverySelectors will also limit the configurations (like Gateway, VirtualService, DestinationRule, Ingress, etc.) that can be processed by pilot.</td>
```

Reply:
This file is updated upstream and should not be changed in this PR. Changes will be overridden overnight.

Reply:
Thanks for your review. Compared with the previous version, I have restored the document.